sbnanaobj/StandardRecord/SRBlipHitClust.h

```diff
@@ -0,0 +1,53 @@
+////////////////////////////////////////////////////////////////////////
+// \file    SRBlipHitClust.h
+// \brief   SRBlipHitClust struct contains information on hit clusters on each plane used to form a SRBlip
+// \author  $Author: jmclaughlin2@illinoistech.edu
+////////////////////////////////////////////////////////////////////////
+#ifndef SBNANAOBJ_STANDARDRECORD_SRBLIPHITCLUST_H
+#define SBNANAOBJ_STANDARDRECORD_SRBLIPHITCLUST_H
+#include <map>
+#include <set>
```

Comment:
Unused, but `vector` is:
```suggestion
#include <vector>
```